src/mockoptimizer.jl

```diff
@@ -240,7 +240,7 @@ function fakesolve(instance::MockLinQuadOptimizer)
     instance.inner.quadratic_primal_solution = unload(instance.quadratic_primal_solution_stored, instance.inner.quadratic_primal_solution)
     instance.inner.quadratic_dual_solution = unload(instance.quadratic_dual_solution_stored, instance.inner.quadratic_dual_solution)
 
-    nothing
+    return nothing
```

Comment:
Style nit: just `return` instead of `return nothing`

Reply:
where is that in the style guide?
I don't like `return` alone...